### DISCUSSION TITLE
Deploy en heroku

### DISCUSSION THREAD

**Discussion by IanMartin94** [dds-jv/foro#105] [Consultas de herramientas] [answered]
Buenas tardes,

Estamos con inconvenientes cuando queremos conectar heroku con nuestro repo de git.

Nos arroja el mensaje "Admin access to repository required".

![image](https://user-images.githubusercontent.com/11914184/100771780-94abad00-33dd-11eb-9111-2a83a1b1a057.png)

Puede ser que nos tengan que dar permisos de admin al repo?

Desde ya muchas gracias.

Saludos !

**Comment by gastonprieto** [comment]
Buenas @IanMartin94 !

Creo que eso es necesario porque lo estas conectando directo a Github. Fijate con este [guía](https://docs.google.com/document/d/1ezGI4s_Jy13JI7Mn-eMXUOUkq7SmVkIYN74hW__6hXI/edit?usp=drive_web&ouid=105127262669791671578) si te evitas eso

**Comment by IanMartin94** [comment, marked as answer]
Buenisimo, ahí pudimos deployar. Muchas gracias !!
